lms/djangoapps/mobile_api/badges/urls.py

```diff
@@ -0,0 +1,37 @@
+"""
+URLs for badges API
+"""
+from django.conf.urls import patterns, url
+from django.conf import settings
+
+from mobile_api.badges.views import BadgeClassDetail, UserBadgeAssertions
+from openedx.core.djangoapps.user_api.urls import USERNAME_PATTERN
+
+BADGE_PATTERN = r'(?P<issuing_component>[-\w]*)/(?P<slug>[-\w]+)'
+
+BADGE_PATTERN_COURSE = BADGE_PATTERN + '/course/' + settings.COURSE_ID_PATTERN
+
+BADGE_PATTERN_WILDCARD = BADGE_PATTERN + '/course/(?P<course_id>[*])'
+
+urlpatterns = patterns(
```

Comment:
For example, an XBlock may award a particular badge. We might want to get all instances of the badges it awards, which it might do once per course. OR we might want to get all the badges a user got while working on a course, but if the XBlock's badge is not course scoped, it wouldn't be included.